test/jdk/javax/swing/JFileChooser/8080628/bug8080628.java

```diff
@@ -33,7 +33,17 @@
  * @test
  * @bug 8080628
  * @summary No mnemonics on Open and Save buttons in JFileChooser.
- * @author Alexey Ivanov
+ * @requires os.family != "linux"
+ * @modules java.desktop/sun.swing
+ * @run main bug8080628
+ */
+
+/*
+ * @test
+ * @bug 8080628
+ * @key headful
+ * @summary No mnemonics on Open and Save buttons in JFileChooser.
+ * @requires os.family == "linux"
```

Comment:
Looks like jtreg tags were accidentally duped. Did you mean to only replace the author tag with the requires tag?

Reply:
@alisenchung Multiple jtreg headers are added to run with different configuration
The 1st one runs in headless mode on platforms other than linux  `@requires os.family != "linux"`
The 2nd one runs in headful mode on linux `@requires os.family == "linux"`

Reply:
@alisenchung As @honkar-jdk explained this is to run different configuration for different platforms.